app/models/speaker.rb

```diff
@@ -20,6 +23,10 @@ def email
     speaker_email.present? ? speaker_email : user.try(:email)
   end
 
+  def more_than_one?(speaker)
+    speaker.program_session.speakers.count > 1
```

Comment:
I feel like a method that tells you whether a program session has more than one speaker should probably be defined on the program session itself. So in the view you could be like speaker.program_session.multiple_speakers?